changelogs/unreleased/fix-incorrect-print-of-msgpack-field.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/msgpack
+
+* Fix incorrect printing of the msgpack field
+  with the MP_EXT type.
```

Comment:
Nit: the line length limit is 80 symbols. I'd understood if you used 72 symbols, but not 45? I notice that you often trim lines too short. Please don't.